fbc-integration-test/components/fbs-4.13.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Component
+metadata:
+  name: 413-fbc-test
+  annotations:
+    pipelinesascode: '1'
+    appstudio.openshift.io/pac-provision: "request"
+spec:
+  application: fbc-e2e-test
```

Comment:
the application name here should match with the application name in the application.yaml file.

Reply:
this all irrelevant and removed, since we are running on RHTAP stage so CRs should be deployed on Stage backend